junit-jupiter-api/src/main/java/org/junit/jupiter/api/extension/AnnotatedElementContext.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2015-2023 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.jupiter.api.extension;
+
+import static org.apiguardian.api.API.Status.EXPERIMENTAL;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.AnnotatedElement;
+import java.util.List;
+import java.util.Optional;
+
+import org.apiguardian.api.API;
+
+@API(status = EXPERIMENTAL, since = "5.10")
+public interface AnnotatedElementContext {
+
+	/**
+	 * Get the {@link AnnotatedElement} for this context.
+	 *
+	 * <h4>WARNING</h4>
+	 * <p>When searching for annotations on the parameter in this context,
```

Comment:
It should not mention "parameter" here since it can be any annotated element.

Reply:
Whoops, missed that one! Thanks.